blockmanager.go

```diff
@@ -432,6 +486,17 @@ func (b *blockManager) handleDonePeerMsg(peers *list.List, sp *ServerPeer) {
 	}
 
 	log.Infof("Lost peer %s", sp)
+}
+
+// handleDonePeerMsg deals with peers that have signalled they are done.  It
+// removes the peer as a candidate for syncing and in the case where it was the
+// current sync peer, attempts to select a new best peer to sync from.  It is
+// invoked from the syncHandler goroutine.
+func (b *blockManager) handleDonePeerMsg(peers *list.List, sp *ServerPeer) {
+	// Remove the peer from the list of candidate peers.
+	b.removeDonePeerFromList(peers, sp)
+
+	log.Infof("Lost peer %s", sp)
```

Comment:
There's no "Msg" in any of the arguments here. The function signature does not give good indication of what this is doing.

Reply:
I did not write that function signature. It was approved and merged into the codebase and it is currently present there.

Reply:
for consistency, change the function signature for this function to take `donePeerMsg` and change the callsite to pass the entire message in. 

Alternatively, rename the function to better describe what it's doing. The latter is more ambitious given that it is hard to concisely describe the process of both removing the peer and searching for a new sync peer. I do not recommend doing this unless you have the patience to make this refactor much larger.